app/models/user.rb

```diff
@@ -1,5 +1,8 @@
 class User < ApplicationRecord
```

Comment:
Please, kindly add to user model relationship with hotel and reservation models.

has_many :hotels
has_many :reservations